src/launcher/standalone_launcher/README.md

```diff
@@ -0,0 +1,157 @@
+# Reactive Trader Client
```

Comment:
This README is not relevant to the launcher. We need to write a specific README for the launcher. Please delete anything in here that is only related to the client app.